Apsitvarkom.UnitTests/Api/Controllers/PollutedLocationControllerTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Apsitvarkom.Api.Controllers;
+using Apsitvarkom.DataAccess;
+using Apsitvarkom.Models.DTO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+
+namespace Apsitvarkom.UnitTests.Api.Controllers;
+
+public class PollutedLocationControllerTests
+{
+    private PollutedLocationController m_controller;
+    private Mock<IPollutedLocationDTORepository> m_repository;
+
+    private readonly IEnumerable<PollutedLocationDTO> PollutedLocationDTOs = new List<PollutedLocationDTO>
```

Comment:
This is just a field now and no matter how many times you get the value of this field, the `Id`s will be constant – field value is assigned when this class (`PollutedLocationControllerTests`) is created. If you wanted the `Id` to be a different value each time you retrieve this `PollutedLocationDTOs`, you would need to turn this **field** into a **property with a getter**. You can use the short syntax by simply changing the assign `=` symbol into the expression symbol `=>`

Reply:
This is not something that you need to fix (since you don't really care about the `Id` field), just a note on the behavior

Reply:
Cool, will have that in mind later. Thanks.